en/advanced_config/imu_factory_calibration.md

```diff
@@ -0,0 +1,30 @@
+# IMU Factory Calibration
+
+PX4 OEM manufacturers can perform an IMU factory calibration in order to store persistent values for the accelerometer, gyroscope and magnetometer biases into persistent EEPROM.
+This ensures that end users can always reset vehicle configurations and tuning to a safe, flying state.
+
+This procedure will write the following parameters to `/fs/mtd_caldata`: [CAL_ACC*](../advanced_config/parameter_reference.md#CAL_ACC0_ID), [CAL_MAG*](../advanced_config/parameter_reference.md#CAL_MAG0_ID), [CAL_GYRO*](../advanced_config/parameter_reference.md#CAL_GYRO0_ID).
+This data will then be used when the parameters are set (or reset) to their default values.
+
+:::warning
+This feature is dependent on the availability of storage at `/fs/mtd_caldata`.
```

Comment:
So if the board has EEPROM and the board config is properly set up, the directory `/fs/mtd_caldata` should automatically be created, right? No user intervention required?

Reply:
I very much doubt that creation or not of the file would be board-specific. I've updated the text to note this.